docker-compose.yml

```diff
@@ -63,27 +66,26 @@ services:
       # /tmp is required by php for session save this should be reworked to /services/run/tmp
       - "/tmp:uid=20211,gid=20211,mode=1700,rw,noexec,nosuid,nodev,async,noatime,nodiratime"
     environment:
-      LISTEN_ADDR: 0.0.0.0                          # Listen for connections on all interfaces
-      PORT: ${PORT}                                 # Application port
-      ALWAYS_FRESH_INSTALL: ${ALWAYS_FRESH_INSTALL} # Set to true to reset your config and database on each container start
-      NETALERTX_DEBUG: 0                            # 0=kill all services and restart if any dies. 1 keeps running dead services.
-      TZ: ${TZ}                                     # Timezone, e.g. Europe/Paris
-      # APP_CONF_OVERRIDE={"SCAN_SUBNETS":"['192.168.1.0/24 --interface=eth1']","GRAPHQL_PORT":"20223","UI_theme":"Light"} # (optional) app.conf settings override
-      # LOADED_PLUGINS=["DHCPLSS","PIHOLE","ASUSWRT","FREEBOX"] # (optional) default plugins to load
+      LISTEN_ADDR: ${LISTEN_ADDR:-0.0.0.0}                      # Listen for connections on all interfaces
+      PORT: ${PORT:-20211}                               # Application port
+      APP_CONF_OVERRIDE: ${GRAPHQL_PORT:-20212}                       # GraphQL API port
```

Comment:
Thanks! Unfortunately still not correct :) APP_CONF_OVERRIDE is a JSON object,so I assume it should be: 

APP_CONF_OVERRIDE=${APP_CONF_OVERRIDE:-{"GRAPHQL_PORT":"20212"}}

not sure about the `-`

Reply:
`APP_CONF_OVERRIDE: ${GRAPHQL_PORT:-20212}`  sets  NetAlertX env variable APP_CONF_OVERRIDE to 20212 unless GRAPHQL_PORT is set. 

* no variable: `APP_CONF_OVERRIDE=20212`
* Variable: `APP_CONF_OVERRIDE=${GRAPHQL_PORT}`

The `:-` annotation sets the fall back.  Tell me how it should be.

Reply:
Then I think this is the right syntax:

`APP_CONF_OVERRIDE=${APP_CONF_OVERRIDE:-{"GRAPHQL_PORT":"20212"}}`

note APP_CONF_OVERRIDE is a JSON object, e.g. `{"GRAPHQL_PORT":"20212"}` or `{"GRAPHQL_PORT":"20212","UI_LANG":"English (en_us)"}`

no variable should result into:

`APP_CONF_OVERRIDE={"GRAPHQL_PORT":"20212"}`

hope that makes sense